webpack/components/AnsibleHostDetail/components/JobsTab/JobsTabHelper.js

```diff
@@ -97,3 +109,6 @@ export const readableCron = (cron = '') => {
 
   return 'custom';
 };
+
+export const readablePurpose = (purpose = '') =>
+  purpose.match(/hostgroup/) ? __('(from host group)') : '';
```

Comment:
This will be called on the host and hostgroup page, right? Can we do it somewhat explicit?
This works, it's just ambigous to have `from host group` on the hostgroup page.